ci/prow/config_knative.yaml

```diff
@@ -173,6 +177,10 @@ presubmits:
       dot-dev: true
 
   knative/caching:
+    - repo-settings:
+      go112-branches:
+      - release-0.9
```

Comment:
caching does not have these two releases. Will leaving it empty work for the config generator?